src/nv_ingest/schemas/vdb_task_sink_schema.py

```diff
@@ -63,16 +63,16 @@ def build_default_milvus_config(embedding_size: int = 1024) -> typing.Dict[str,
                     description="Embedding vectors",
                     dim=embedding_size,
                 ).to_dict(),
-                pymilvus.FieldSchema(
-                    name="source",
-                    dtype=pymilvus.DataType.JSON,
-                    description="Source document and raw data extracted content",
-                ).to_dict(),
-                pymilvus.FieldSchema(
-                    name="content_metadata",
-                    dtype=pymilvus.DataType.JSON,
-                    description="Content metadata",
-                ).to_dict(),
+                # pymilvus.FieldSchema(
```

Comment:
removed these fields because they are JSON is not compatible with parquet files. If we were to keep this data, when we go to upload, the ingestion blows up because milvus expect json data but actually gets structured data from arrow.

Reply:
@mpenn I'm not familiar enough with the lineage here, could you comment?

Reply:
The lineage for this was to provide content in an intuitive structure to support the RAG workflow behind the [multimodal-pdf-data extraction blueprint](https://build.nvidia.com/nvidia/multimodal-pdf-data-extraction-for-enterprise-rag/blueprintcard). If we decide to move from this structure, it will likely require some blueprint re-work. Not impossible but would require some re-work.

Reply:
Still need further guidance on how we want to handle this schema change for VDB entries.

Reply:
Waiting on a decision from @randerzander on this; would definitely prefer a work around if it is feasible, as this is almost certainly a breaking change for some workflows.